polytope/polytope.py

```diff
@@ -201,19 +201,40 @@ def __copy__(self):
         P.fulldim = self.fulldim
         return P
 
+    def contains(self, point, abs_tol=ABS_TOL):
+        """Return True if polytope contains points.
+
+        @type point: array_like, 1d array, or 2d array (of column vectors)
+
+        @rtype: bool, 1d array
+
+        See Also
+        ========
+        L{is_inside}
+        """
+        if not isinstance(point, np.ndarray):
```

Comment:
I am inclined to suggest avoiding `isinstance` and type casting, and instead require that the user explicitly pass a suitable object. Creating new objects within a method that checks a condition may be unexpected.

This approach is already present in the method `__contains__`. It may be worth considering removing it from there.